legend-engine-ide-lsp-server/src/main/java/org/finos/legend/engine/ide/lsp/server/LegendTextDocumentService.java

```diff
@@ -95,56 +94,67 @@ class LegendTextDocumentService implements TextDocumentService
     @Override
     public void didOpen(DidOpenTextDocumentParams params)
     {
+        LegendServerGlobalState globalState = this.server.getGlobalState();
+        this.server.checkReady();
+        TextDocumentItem doc = params.getTextDocument();
+        String uri = doc.getUri();
+        LOGGER.debug("Opened {} (language id: {}, version: {})", uri, doc.getLanguageId(), doc.getVersion());
```

Comment:
I would either move the logging back inside runPossiblyAsync or add logging there, since that's when the file open operation will actually happen.